### PR TITLE
S3 Lambda example

### PR DESCRIPTION
**Pull request by jaichopra**

### CODE REVIEW

File: rust/src/lib.rs [hunk 1]
```diff
@@ -0,0 +1,14 @@
+pub fn add(left: usize, right: usize) -> usize {
```
Comment (changhiskhan) [inline rust/src/lib.rs:1]: ?

File: docs/src/examples/langchain.md [hunk 2]
```diff
@@ -0,0 +1,138 @@
+# Code Documentation Q&A Bot 2.0
```
Comment (changhiskhan) [inline docs/src/examples/langchain.md:1]: this all references lance format

File: rust/src/lib.rs (hunk 1 above)
Comment (jaichopra) [reply rust/src/lib.rs:1]: Hmm residual code from a rebase, I'll clean up

File: docs/src/examples/langchain.md (hunk 2 above)
Comment (jaichopra) [reply docs/src/examples/langchain.md:1]: I'll remove this example and fix in another PR

**Review by changhiskhan — APPROVED** [review]

**Pull request merged**